mysql-integration/src/main/java/com/dataloom/integrations/dataintegration/DataIntegration.java

```diff
@@ -0,0 +1,81 @@
+
+public class DataIntegration {
+    // Logger is used to output useful debugging messages to the console
+    private static final Logger logger = LoggerFactory.getLogger( DataIntegration.class );
+
+    // Set your Entity Set Name, Type, and Primary Key
+    public static String            ENTITY_SET_NAME = "testmysql";
+    public static FullQualifiedName ENTITY_SET_TYPE = new FullQualifiedName( "sample.mysql" );
+    public static FullQualifiedName ENTITY_SET_KEY1 = new FullQualifiedName( "general.firstname" );
+    public static FullQualifiedName ENTITY_SET_KEY2 = new FullQualifiedName( "general.lastname" );
+    public static FullQualifiedName ENTITY_SET_KEY3 = new FullQualifiedName( "general.dob" );
+
+    public static void main( String[] args ) throws InterruptedException {
+
+        final String path = args[ 0 ];
+        final String username = args[ 1 ];
+        final String password = args[ 2 ];
+        final SparkSession sparkSession = MissionControl.getSparkSession();
+        final String jwtToken = MissionControl.getIdToken( username, password );
+        logger.info( "Using the following idToken: Bearer {}", jwtToken );
+
+        Dataset<Row> payload = sparkSession.read()
+                .format( "jdbc" )
+                .option( "url", "jdbc:mysql://[host]:[port]/[database]" )
+                .option( "driver", "com.mysql.jdbc.Driver" )
+                .option( "dbtable", "people" )
+                .option( "user", "root" )
+                .option( "password", "mysqlpassword" )
+                .load();
+
+        Flight flight = Flight.newFlight()
+                .addEntity( ENTITY_SET_TYPE )
+                .to( ENTITY_SET_NAME )
+                .key( ENTITY_SET_KEY1, ENTITY_SET_KEY2, ENTITY_SET_KEY3 )
+                .addProperty( new FullQualifiedName( "general.firstname" ) )
+                .value( row -> row.getAs( "first_name" ) ).ok()
+                .addProperty( new FullQualifiedName( "general.uuid" ) )
+                .value( row -> row.getAs( "uuid" ) ).ok()
+                .addProperty( new FullQualifiedName( "general.dob" ) )
+                .value( row -> standardizeDate( row.getAs( "dob" ) ) ).ok()
+                .addProperty( new FullQualifiedName( "general.ssn" ) )
+                .value( row -> row.getAs( "ssn" ) ).ok()
+                .addProperty( new FullQualifiedName( "general.ethnicity" ) )
+                .value( row -> row.getAs( "ethnicity" ) ).ok()
+                .addProperty( new FullQualifiedName( "general.gender" ) )
+                .value( row -> row.getAs( "sex" ) ).ok()
+                .addProperty( new FullQualifiedName( "general.race" ) )
+                .value( row -> row.getAs( "race" ) ).ok()
+                .addProperty( new FullQualifiedName( "general.lastname" ) )
+                .value( row -> row.getAs( "last_name" ) ).ok()
+                .ok()
+                .done();
+
+        Shuttle shuttle = new Shuttle( jwtToken );
```

Comment:
I believe the default environment is production. We should use `Environment.LOCAL` for test integrations.